packages/cozy-mespapiers-lib/src/components/Home/helpers.js

```diff
@@ -1,11 +1,51 @@
-// should be in cozy-client : https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/models/document/documentTypeDataHelpers.js
-export const hasItemByLabel = (theme, label) =>
-  theme.items.some(item => item.label === label)
+import Fuse from 'fuse.js'
 
-export const filterPapersByTheme = (files, theme) => {
-  if (!theme) return files
+const fuse = new Fuse([], {
+  findAllMatches: true,
+  threshold: 0.3,
+  ignoreLocation: true,
+  ignoreFieldNorm: true,
+  keys: [
+    {
+      name: 'name'
+    },
+    {
+      name: 'metadata.qualification.name'
+    }
+  ]
```

Comment:
Does the keys have to be the same? It seems weird to have the same key both for the file name and the (translated) qualification label.

Reply:
c'est le fonctionnement de fusejs. Tous les champs sur lesquels on veut faire la recherche sont dans des objects dont le nom du champ est dans une clé `name`. Par contre la clé dans l'object n'est pas la même bien sûr

Reply:
I beg to differ: https://fusejs.io/demo.html and https://fusejs.io/examples.html#search-string-array
Je ne vois nulle part le besoin d'une clé `name`, où vois-tu ça ? :thinking:

Reply:
J'avais repris le cas d'usage dans Contacts qui s'appuyait sur https://fusejs.io/examples.html#default-weight je suppose.

Il n'y a pas de notion de poids ici, j'ai donc simplifié.

Si besoin de savoir le pourquoi du comment en creusant le code de la lib parce que la doc n'est pas assez explicite, je pense qu'on peut faire ça dans un deuxième temps sans bloquer la PR, non ?

Reply:
=> https://github.com/krisk/Fuse/blob/master/src/tools/KeyStore.js#L53

Reply:
Il y a donc besoin de `name` quand on veut utiliser la recherche pondérée et qu'on a besoin d'un objet, mais autrement on peut passer directement la clé en string, e.g. `keys: [toto, tutu]`. 
En tout cas, ce n'est pas bloquant